backend/app/api/routes/content.py

```
@@ -1064,39 +1114,54 @@ async def stream_analysis():
 
 
 async def _stream_content_analysis_ai_sdk(
-    content_item: ContentItem,
+    content_item_id: uuid.UUID,  # 使用ID而不是对象
```

Ruff (0.11.9)

Unused function argument: `content_item_id`

(ARG001)